db/migrate/20230227131731_create_job_preferences_locations.rb

```diff
@@ -0,0 +1,59 @@
+require 'geocoding'
+
+class CreateJobPreferencesLocations < ActiveRecord::Migration[7.0]
+  class JobPreferences < ActiveRecord::Base
+    has_many :location_records, class_name: 'JobPreferencesLocations'
+  end
+
+  class JobPreferencesLocations < ActiveRecord::Base
```

Comment:
This is to prevent changes in the model from affecting the migration. If we ever decide to delete any of the records with future migration, this migration holds its own copy of the record in a form that is required for data migration.

I was not able to isolate Geocoding and LocationPolygon, but it's probably ok since they both seem to be pretty stable implementation